src/content/enterprise/install/releases.mdx

```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.0.0
+September 1, 2022
+
+### Removal Notice
+- Kubernetes [1.19](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md) is no longer supprted.
+
+### Deprecation Notice
+- Support for Kubernetes [1.20](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md) has been derprecated and will be removed next release.
+
+### Features
+- License enforcements changed to allow for easier adoption.
```

Comment:
In general, I think we should describe this in terms of the user benefit. 

But given the sensitive nature of this one (it could lead to abuse), I think it's better not to mention it.

Reply:
- [ ] Remove this.